AsyncDisplayKit/ASButtonNode.mm

```diff
@@ -491,10 +491,17 @@ - (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
     spec = [ASInsetLayoutSpec insetLayoutSpecWithInsets:contentEdgeInsets child:spec];
   }
   
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
   if (CGSizeEqualToSize(self.preferredFrameSize, CGSizeZero) == NO) {
-    stack.sizeRange = ASRelativeSizeRangeMakeWithExactCGSize(self.preferredFrameSize);
+#if DEBUG
```

Comment:
We should keep preferredFrameSize around for deprecation, but end usage of it internally to make it easier to deprecate and to set an example of how to use the new API where we used to use PFS.